templates/default/rt/rtrc.erb

```diff
@@ -0,0 +1,3 @@
+server http://rtlocal
```

Comment:
Does this need to be different depending on how this gets deployed in production?

Reply:
This can have an issue if we are wanting to deploy multiple RT instances on the same server. Since this was last implemented, the "domain" tied to the site has changed to an attribute. So, it should theoretically allow for multiple RT sites on a single VM. (Once we turn this into a resource)

Reply:
I think it's safe to assume there will only be one instance of RT per system.